### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception Operator called default onErrorDropped reactor.core.Exceptions$ErrorCallbackNotImplemented: reactor.core.Exceptions$OverflowException: Could not emit tick 624 due to lack of requests (interval doesn't support small downstream requests that replenish slower than the ticks)

### ISSUE THREAD

**Issue by samtorqueits** [microsoft/azure-tools-for-java#8205]
IntelliJ build version: 2023.3.4 IU-233.14475.28
OS: Windows 11
JDK: JetBrains s.r.o. 17.0.10
Plugin version: 3.86.0-2023.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,8,8,1920x1032,layout=com.intellij.openapi.wm.impl.IdeRootPane$MyRootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=Operator called default onErrorDropped, throwable=reactor.core.Exceptions$ErrorCallbackNotImplemented: reactor.core.Exceptions$OverflowException: Could not emit tick 624 due to lack of requests (interval doesn't support small downstream requests that replenish slower than the ticks)
Caused by: reactor.core.Exceptions$OverflowException: Could not emit tick 624 due to lack of requests (interval doesn't support small downstream requests that replenish slower than the ticks)
```

**Comment by wangmingliang-ms** [comment]
fixed in 3.87.0, please upgrade to the latest version.

**Comment by wangmingliang-ms** [comment]
fixed in later versions, please upgrade.
